app/src/main/java/org/schabi/newpipe/player/gesture/MainPlayerGestureListener.kt

```diff
@@ -193,10 +194,16 @@ class MainPlayerGestureListener(
         isMoving = true
 
         // -- Brightness and Volume control --
-        val isBrightnessGestureEnabled = PlayerHelper.isBrightnessGestureEnabled(player.context)
-        val isVolumeGestureEnabled = PlayerHelper.isVolumeGestureEnabled(player.context)
+        var isBrightnessGestureEnabled = PlayerHelper.isBrightnessGestureEnabled(player.context)
+        var isVolumeGestureEnabled = PlayerHelper.isVolumeGestureEnabled(player.context)
+        var displaySide = DisplayPortion.LEFT_HALF
+        val sidesSwitched = PreferenceManager.getDefaultSharedPreferences(player.context)
+            .getBoolean(R.string.switch_gesture_sides_key.toString(), false)
+        if (sidesSwitched) {
+            displaySide = DisplayPortion.RIGHT_HALF
+        }
```

Comment:
I'd do:
```suggestion
        val brightnessSide = if (PreferenceManager.getDefaultSharedPreferences(player.context)
            .getBoolean(R.string.switch_gesture_sides_key.toString(), false)) DisplayPortion.RIGHT_HALF
            else DisplayPortion.LEFT_HALF
```

Reply:
I made this change but now the function doesn't work anymore. The sides ar not switched when the toggle is switched on.